### ISSUE TITLE
IfcGridAxis and IfcDimension position is not saved with BlenderBIM

### ISSUE THREAD

**Issue by zhukoven** [IfcOpenShell/IfcOpenShell#4881]
I'm facing an issue with IfcGridAxis and IfcDimensions that are not saved together with the other geometry and data in the IFC file. These elements are being reset to their original location every time that I reopen my ifc file with BlenderBIM. 

It's relatively easy to reproduce:
1. Add grids/dimensions
2. Select some grids/dimensions, then enter the Edit mode with `TAB`
3. Move the grids/dimensions to desired position and save the IFC file
4. Re-open recently saved IFC and check elements' position.

I'm using the latest "stable" BlenderBIM release (v0.0.240602-47704b1) in Blender 4.1.1

**Comment by theoryshaw** [comment]
seems related: https://github.com/IfcOpenShell/IfcOpenShell/issues/4769

**Comment by zhukoven** [comment]
@theoryshaw indeed these are related things. I also spent some time trying to workaround the issue, and I think that I've found solution:
<img width="1694" alt="image" src="https://github.com/IfcOpenShell/IfcOpenShell/assets/27201628/c20e3816-d0d8-451d-a017-fb13efd57802">
It seems that the grids lose their position when I try moving them while in the Edit mode. However, when I move the grids in the Object mode, they do save their position. 

Here are the steps to try:
1. Staying in the object mode, select the Grid line
2. On the Item palette, unblock XY transforms (see the screenshot)
3. Move the gridline to desired position
4. Lock transforms
5. Save IFC and re-open to see if the grids stay in place.

I haven't yet tested this workflow with dimensions though.
